commerce_coordinator/settings/test.py

```diff
@@ -40,6 +40,145 @@
     }
 }
 
+CC_SIGNALS = {
+    # DEMO: This configuration is just for proof-of-concept and can
+    # be removed once we have real signals
+    'commerce_coordinator.apps.core.signals.test_signal': [
```

Comment:
If we are removing these configurations from base, why are we adding them here? Are we simply ensuring references to these signals in the rest of the codebase have somewhere to point to to reduce errors, until we've cleaned up these signals in their respective apps? Or was this missed?

Is there also a ticket for future work to refactor the codebase to remove these unnecessary filters and signal functions?

Reply:
Code coverage drops if the code is there and not tested... We would go from like 99% to like 40% coverage... we can guy them after release when we remove all the titan work.

Reply:
There isnt a ticket, ill make one.